OSDOCS-20684: Add 4.22.5 z-stream release notes (#115396)
openshift/openshift-docs: modules/zstream-4-22-5.adoc

// Module included in the following assemblies:
//
// * release_notes/ocp-4-22-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-22-5_{context}"]
= RHSA-2026:37585 - {product-title} {product-version}.5 bug fix and security update

Issued: 14 July 2026

[role="_abstract"]
{product-title} release {product-version}.5 is now available. The list of fixed issues that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:37585[RHSA-2026:37585] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2026:37583[RHBA-2026:37583] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.22.5 --pullspecs
----

[id="zstream-4-22-5-fixed-issues_{context}"]
== Fixed issues

> * Before this update, when deploying a hosted cluster on AWS in non-commercial ISO or classified regions, the Cluster Network Operator (CNO) assumed a maximum transmission unit (MTU) of 9001 for hosted cluster nodes. Because some of these regions do not support jumbo frames at this MTU value, and the setting could not be overridden during cluster creation, networking for hosted clusters in these regions could fail to configure correctly. With this release, you can specify an OVN-Kubernetes tunnel interface MTU between 576 and 9216 at cluster creation time. You can configure this setting by using the optional, immutable `mtu` field in the `ovnKubernetesConfig` configuration of the `HostedCluster` custom resource (CR) or by using the `--ovn-kubernetes-mtu` hosted control planes CLI flag. As a result, you can successfully deploy hosted clusters in AWS regions that do not support jumbo frames, and the CNO applies the correct MTU value. (link:https://redhat.atlassian.net/browse/OCPBUGS-77078[OCPBUGS-77078])

Reviewer: ```suggestion
* Before this update, when deploying a hosted cluster on AWS in non-commercial ISO or classified regions, the Cluster Network Operator (CNO) assumed a maximum transmission unit (MTU) of `9001` for hosted cluster nodes. Because some of these regions do not support jumbo frames at this MTU value, and the setting could not be overridden during cluster creation, networking for hosted clusters in these regions could fail to configure correctly. With this release, you can specify an OVN-Kubernetes tunnel interface MTU between `576` and `9216` at cluster creation time. You can configure this setting by using the optional, immutable `mtu` field in the `ovnKubernetesConfig` configuration of the `HostedCluster` custom resource (CR) or by using the `--ovn-kubernetes-mtu` hosted control planes CLI flag. As a result, you can successfully deploy hosted clusters in AWS regions that do not support jumbo frames, and the CNO applies the correct MTU value. (link:https://redhat.atlassian.net/browse/OCPBUGS-77078[OCPBUGS-77078])
```

> * Before this update, when you filtered projects on the *Projects* list page in the {product-title} web console, the filter matched only the project `metadata.name` value and not the display name stored in the `openshift.io/display-name` annotation. As a consequence, typing a project's display name in the filter toolbar returned no results even when a matching project existed. With this release, the *Projects* list filter includes the display name in its matching logic for both fuzzy and exact search modes. As a result, you can filter projects by either their technical name or their display name. (link:https://redhat.atlassian.net/browse/OCPBUGS-90495[OCPBUGS-90495])

Reviewer: ```suggestion
* Before this update, when you filtered projects on the *Projects* list page in the {product-title} web console, the filter matched only the project `metadata.name` value and not the display name stored in the `openshift.io/display-name` annotation. As a consequence, typing a project display name in the filter toolbar returned no results even when a matching project existed. With this release, the *Projects* list filter includes the display name in its matching logic for both fuzzy and exact search modes. As a result, you can filter projects by either their technical name or their display name. (link:https://redhat.atlassian.net/browse/OCPBUGS-90495[OCPBUGS-90495])
```
remove possessive's when possible (IBM)

* Before this update, on Secure Boot-enabled clusters with nodes originally installed on {product-title} 4.15 or earlier, the Machine Config Operator (MCO) verified the shim version in the node operating system image rather than in the EFI System Partition (ESP). Nodes that were updated in place could have outdated shim and GRand Unified Bootloader (GRUB) binaries on the ESP while the operating system image contained newer versions. As a consequence, the MCO incorrectly skipped the boot loader update workaround for those nodes during a cluster update, and the nodes could fail to boot after the update. With this release, when a node runs Red Hat Enterprise Linux (RHEL) 9.6 or later, the MCO runs a node-level boot loader update by using `bootupctl` before it pulls the new node image container, and falls back to the existing container-based update path if the node-level update fails. As a result, nodes with stale boot loader files on the ESP are updated correctly and boot successfully after updating to this release. (link:https://redhat.atlassian.net/browse/OCPBUGS-93743[OCPBUGS-93743])

> * Before this update, when Prometheus compacted time series data, it attempted to enable direct I/O on files that were already open. Some file systems, such as IBM Storage Scale, do not support this operation. As a consequence, compaction failed with a `cannot enable Direct IO: invalid argument` error, Prometheus could not write chunks to disk, and memory usage in Prometheus pods increased until the pods hit their memory limits and restarted or affected other workloads on the node. With this release, Prometheus detects when a file system does not support direct I/O on open files and handles the condition gracefully. As a result, Prometheus writes chunks to disk and completes compaction successfully on these storage back ends. (link:https://redhat.atlassian.net/browse/OCPBUGS-93919[OCPBUGS-93919])

Reviewer: ```suggestion
* Before this update, when Prometheus compacted time series data, it attempted to enable direct input/output on files that were already open. Some file systems, such as IBM Storage Scale, do not support this operation. As a consequence, compaction failed with a `cannot enable Direct IO: invalid argument` error. Prometheus could not write chunks to disk, and memory usage in Prometheus pods increased until the pods hit their memory limits and restarted or affected other workloads on the node. With this release, Prometheus detects when a file system does not support direct input/output on open files and handles the condition gracefully. As a result, Prometheus writes chunks to disk and completes compaction successfully on these storage back ends. (link:https://redhat.atlassian.net/browse/OCPBUGS-93919[OCPBUGS-93919])
```
